docker/checkout.bash

```diff
@@ -35,10 +35,16 @@ while [[ $# -gt 0 ]]; do
       esac
 done
 
+cd ~/src
+####Add dependencies#######
+
+# novatel driver 
+sudo git clone https://github.com/novatel/novatel_oem7_driver.git --branch ros2-dev 3055e220bb9715b59c3ef53ab0aba05a495d9d5c
```

Comment:
I'm fairly certain this command is checking out this repo into a folder named 3055e220bb9715b59c3ef53ab0aba05a495d9d5c
To checkout the specific commit you would first cd into the folder then do a git checkout 3055e220bb9715b59c3ef53ab0aba05a495d9d5c